selfdrive/car/gm/interface.py

```diff
@@ -280,7 +280,7 @@ def _update(self, c):
     if below_min_enable_speed and not (ret.standstill and ret.brake >= 20 and
                                        self.CP.networkLocation == NetworkLocation.fwdCamera):
       events.add(EventName.belowEngageSpeed)
-    if ret.cruiseState.standstill:
+    if ret.cruiseState.standstill and not self.CP.autoResumeSng:
```

Comment:
We only need one change, right? Either this or changing the priorities.

Reply:
I worry that cruiseControl.resume may be noisy, so we will spam the audible min steering speed alert in stop and go.

I haven't personally tried this, so maybe it's a non-issue. Have you been driving on this branch, what is your experience? Is it annoying to hear the alert when you press resume?

Reply:
This isn't doing anything with the CC resume signal, though you are correct that it is noisy. This change would just not show the "press resume" alert if the user is not in fact required to press resume. (Note that currently, no GM cars have working sng on stock; this is more of a preemptive change to the alert for when that becomes a reality.)